GothicVRProject/GothicVR
# Feature/sound volume sliders

# To test
1. Test volume in main menu
2. Load world and test if volume is properly set
3. Restart the game and check if sound volume is properly set

# Testing
